static/js/settings_bots.js

```diff
@@ -582,6 +583,14 @@ export function set_up() {
         $(this).attr("href", generate_zuliprc_uri(bot_id));
     });
 
+    $("#active_bots_list").on("click", "button.open_bots_subscribed_streams", (e) => {
+        e.preventDefault();
+        e.stopPropagation();
+        const bot_id = Number.parseInt($(e.currentTarget).attr("data-user-id"), 10);
+        const bot = people.get_by_user_id(bot_id);
+        user_profile.show_user_profile(bot, 1);
```

Comment:
The `1` parameter is not great for readability; can we pass the key for the tab, `user-profile-streams-tab`, instead?

Reply:
Done.